plugin.image.qwantz/changelog.txt

```diff
@@ -0,0 +1 @@
+1.0.0 Initial Release
```

Comment:
Note: you can also use the news tag for the changelog (for the last version only)